### PR TITLE
Quickbooks

### PR DESCRIPTION
**Pull request by zacmos**

### CODE REVIEW

File: examples/quickbooks/quickbooks.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+// Calls handler when bill is updated
+quickbooksConnector.on({ action: 'Bill/Update' }, async (event, app) => {
+  console.log('Bill/Update event ')
+  console.log(event.id)
+  console.log(event.name)
+  console.log(event.action)
+})
+
+// Calls handler when invoice is created
```
Comment (lagache) [inline examples/quickbooks/quickbooks.js:44]: invoice or bill?

File: examples/quickbooks/quickbooks.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  console.log(event.operation)
+})
+
+async function main() {
+  // The SDK call is inside a setTimeout to allow users clicking the authUri
+  // and complete the authentication process
+
+  // Get company info,
+  setTimeout(async function () {
+    const sdk = await quickbooksConnector.sdk()
```
Comment (lagache) [inline examples/quickbooks/quickbooks.js:58]: the await should prevent the need of setTimeout

File: examples/quickbooks/quickbooks.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+const app = new Reshuffle()
+// Database Store:
+// const pool = new Pool({user: 'RESHUFFLE_DB_USER',
+//   host: 'RESHUFFLE_DB_HOST',
+//   database: 'RESHUFFLE_DB',
+//   password: 'RESHUFFLE_DB_PASS',
+//   port: RESHUFFLE_DB_PORT})
+// const persistentStore = new SQLStoreAdapter(pool, 'reshuffledb')
+
+const persistentStore = new MemoryStoreAdapter()
```
Comment (lagache) [inline examples/quickbooks/quickbooks.js:23]: I think it is the default and we don't even need to do app.setPersistentStore(new MemoryStoreAdapter()).

File: examples/quickbooks/quickbooks.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+const { Reshuffle, SQLStoreAdapter, MemoryStoreAdapter } = require('reshuffle')
+const { QuickbooksConnector } = require('reshuffle-quickbooks-connector')
+const { Pool } = require('pg')
+
+// This example uses reshuffle-quickbooks-connector
+// Code and documentation available on Github: https://github.com/reshufflehq/reshuffle-quickbooks-connector
+
+// Create an API token from your Shopify Admin Page:
```
Comment (lagache) [inline examples/quickbooks/quickbooks.js:8]: Shopify leftovers

**Review by lagache — CHANGES_REQUESTED** [review]

File: examples/quickbooks/quickbooks.js (hunk 4 above)
Comment (ashevat) [reply examples/quickbooks/quickbooks.js:8]: +1 to remove these @zacmos

**Review by ashevat — CHANGES_REQUESTED** [review]

**Review by lagache — APPROVED** [review]

File: examples/quickbooks/quickbooks.js (hunk 2 above)
Comment (zacmos) [reply examples/quickbooks/quickbooks.js:52]: We need the timeout to enable the user clicking the auth link

**Pull request merged**